src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,67 @@
+.dg_summary-list {
+  color: $gray;
+  background-size: cover;
+  background-color: $gray-lightest;
+}
+  .dg_summary-list-items {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+  p {
+    margin-bottom: 0px;
+  }
```

Comment:
@martypowell @sdurphy @danfox01  Looking at the prototype, I think the h3 tag for this component should be `$black`. If this is the case, a style for `h3 {font-color: $black;}` should be nested as a child of `dg_summary-list-items`